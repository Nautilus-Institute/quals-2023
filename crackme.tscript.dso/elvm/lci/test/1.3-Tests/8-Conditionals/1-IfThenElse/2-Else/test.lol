HAI 1.3
	I HAS A var
	var
	O RLY?
		YA RLY
			VISIBLE "1a"
		NO WAI
			VISIBLE "1b"
	OIC
	FAIL
	O RLY?
		YA RLY
			VISIBLE "2a"
		NO WAI
			VISIBLE "2b"
	OIC
	0
	O RLY?
		YA RLY
			VISIBLE "3a"
		NO WAI
			VISIBLE "3b"
	OIC
	0.0
	O RLY?
		YA RLY
			VISIBLE "4a"
		NO WAI
			VISIBLE "4b"
	OIC
	""
	O RLY?
		YA RLY
			VISIBLE "5a"
		NO WAI
			VISIBLE "5b"
	OIC
KTHXBYE
