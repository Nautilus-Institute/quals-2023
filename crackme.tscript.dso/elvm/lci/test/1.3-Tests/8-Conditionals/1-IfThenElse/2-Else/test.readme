This test checks that the operation of the else branch in an if/then/else block
works correctly.
