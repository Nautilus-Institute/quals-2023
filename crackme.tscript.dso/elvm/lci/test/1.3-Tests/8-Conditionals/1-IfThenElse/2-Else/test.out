1b
2b
3b
4b
5b
